src/library_html5.js

```diff
@@ -1016,7 +1016,7 @@ var LibraryJSEvents = {
   emscripten_set_orientationchange_callback_on_thread__sig: 'iiiii',
   emscripten_set_orientationchange_callback_on_thread__deps: ['_registerOrientationChangeEventCallback'],
   emscripten_set_orientationchange_callback_on_thread: function(userData, useCapture, callbackfunc, targetThread) {
-    if (!screen || !screen.addEventListener) return {{{ cDefine('EMSCRIPTEN_RESULT_NOT_SUPPORTED') }}};
+    if (!screen) return {{{ cDefine('EMSCRIPTEN_RESULT_NOT_SUPPORTED') }}};
```

Comment:
This will need revisiting in the future, due to other spec changes; but for now only removed `screen.addEventListener` which was a tautological check.